Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -439,10 +442,12 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
         guard
             let h = high,
             let entry = data?.entryForHighlight(h)
-            else {
+            else
+        {
```

Comment:
indent seems wrong here.

Reply:
It's what Xcode auto indents to. It lines up with the guard instead of the else. I think it makes sense because in a `guard` statement the "else" is syntactic sugar. `guard` is really just a fancy `if` and in `if` statements the "{" lines up the same way.